release/src/main/scripts/build_release_candidate.sh

```diff
@@ -98,7 +105,8 @@ if [[ $confirmation = "y" ]]; then
   echo "2. new rc tag has created in github."
 
   echo "-------------Staging Java Artifacts into Maven---------------"
-  ./gradlew publish -PisRelease --no-daemon
+  gpg --local-user ${SIGNING_KEY} --output /dev/null --sign ~/.bashrc
```

Comment:
Is this for testing that the key is working?

Would not it break if ~/.bashrc does not exist?

Reply:
No. It is to ensure the key is unlocked so gpg-agent will just provide access to the key without requesting for user input within gradle call. As gradle is configured to shell out to gpg cli, streams get broken and no input is possible.

And yes, this will break, if .bashrc does not exist. But the same pattern was used before [1], so I just reused that.

Of course, we might reconsider that.

[1] https://github.com/apache/beam/blob/master/release/src/main/scripts/verify_release_build.sh#L140

Reply:
Fair enough.